vms/platformvm/txs/executor/camino_tx_executor.go

```diff
@@ -1513,6 +1514,11 @@ func (e *CaminoStandardTxExecutor) AddressStateTx(tx *txs.AddressStateTx) error
 	// Calculate new states
 	newStates := states
 	if tx.Remove && (states&statesBit) != 0 {
+		for signerAddresses := range addresses {
```

Comment:
This seems to be wrong, there is an "address" field in TX which should be used for this.....